libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/snapshotter/SnapshotterProcessorTest.kt

```diff
@@ -34,15 +34,6 @@ class SnapshotterProcessorTest {
         ctx = ApplicationProvider.getApplicationContext()
     }
 
-    @Test
-    fun `process action generate snapshot result snapshot unavailable no banner instructions`() {
```

Comment:
Wondering why this test was removed. Isn't a still valid use case? Should this be tested differently (error case) instead of removed? 🤔

Reply:
This test case was added as a result of this [PR](https://github.com/mapbox/mapbox-navigation-android/pull/4099).
But if you look at [this](https://github.com/mapbox/mapbox-navigation-android/pull/4132#issuecomment-799518324) we have agreed to revert the change and hence the test.